src/components/app/userActionRowCTA/index.tsx

```diff
@@ -64,25 +77,35 @@ export const UserActionRowCTAButton = React.forwardRef<
         disabled={!canBeActionedOn}
         ref={ref}
       >
-        <div className="flex items-center gap-4">
+        <div className="flex flex-1 items-center gap-4">
           {state !== 'hidden' && <div className="flex-shrink-0">{getStateUI()}</div>}
-          <div className="hidden flex-shrink-0 md:block">
+          <div className="relative flex h-[80px] w-[80px] flex-shrink-0 items-center justify-center overflow-hidden rounded-xl bg-black lg:h-[100px] lg:w-[100px]">
             <NextImage
-              alt={text}
-              className="rounded-xl"
-              height={100}
-              src={image}
-              style={{ height: 100, width: 100 }}
-              width={100}
+              alt={shortText}
+              className="object-cover lg:h-[80px] lg:w-[80px]"
+              height={60}
+              sizes="(max-width: 768px) 60px, 80px"
+              width={60}
+              {...image}
             />
           </div>
-          <div>
+          <div
+            // Show on Mobile
+            className="sm:hidden"
+          >
+            <div className="mb-1 text-base font-bold lg:text-2xl">{shortText}</div>
+            <div className="text-sm text-gray-500 lg:text-xl">{shortSubtext}</div>
+          </div>
+          <div
+            // Show on Desktop
+            className="max-sm:hidden"
```

Comment:
np: In other places we use the mobile first approach for this, we could unify it to the others
```suggestion
            className="hidden md:block"
```